packages/add-attributes-twig-extension/lib/add-attributes-twig-extension.js

```diff
@@ -15,7 +15,7 @@ function addAttributesTwigExtension(Twig) {
         }
         else {
           // Handle bem() output (pass in exactly the result).
-          if (value.includes('=')) {
+          if (String(value).includes('=')) {
```

Comment:
Is there ever gonna be a case where value could be an object, null or undefined? Because this is going to stringify that. I'm not sure if that's a viable solution. I have no idea what Drupal would do in that case either, as I haven't had a chance to test, but I figured I'd at least ask the question.

Reply:
@WidgetsBurritos Apologies for the very late reply here. I updated the logic in this file to check for several variables types and then handle them accordingly. For Strings, Boolean, and Number type variables we stringify them and pass them along. For the object type differentiate regular objects from arrays and handle arrays as we do now. 
While I definitely think there is room for further improvement I do believe these changes help make the add_attributes function a bit more flexible.